static/docs/commands-reference/init.md

```diff
@@ -1,6 +1,6 @@
 # init
 
-This command initializes a DVC environment in a current Git repository.
+This command initializes a DVC environment in a local Git repository.
```

Comment:
local is confusing term here. What if it's not local to me in a sense that it's located on remote machine - some EC2 instance? Either keep the previous version or just remove it.

Reply:
> What if it's not local to me in a sense that it's located on remote machine?

If I understand correctly, you would be SSH'd into that remote machine so from that terminal session's perspective it's a local repo. "Git repository on your local machine" is the phrase most used in Git docs (see [this search](https://git-scm.com/search/results?search=local%20machine)) but "local repository" is also used a couple times (see [this other search](https://git-scm.com/search/results?search=local%20repo)).

Anyway, I changed it to "Git repository where you're working from" in bc69724. Like?

Reply:
Yep, sounds good. Actually, reading this again, I realized that presence of Git is optional. So, it's more like `in a project directory`. That should be enough.

Reply:
It's not optional.

```consone
$ mkdir empty
$ cd empty
empty$ dvc init
ERROR: failed to initiate dvc - /Users/drbidibombom/empty is not tracked by any
supported scm tool (e.g. git). Use '--no-scm' if you don't want to use any scm.
...
```

Reply:
there is `--no-scm` option for that

Reply:
True! Hmmmm actually this doc needed a bunch of updates once I read it slowly. See 0afdc5d )